soc/arm/atmel_sam/same70/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 zephyr_sources(
   soc.c
   soc_config.c
+  reboot.S
+  sys_arch_reboot.c
```

Comment:
Why are you modifying this for Atmel? Shouldn't it be in `soc/arm/ti_lm3s6965/CMakeLists.txt`?

Reply:
This was a mistake. I'm not sure how my testing didn't catch this ...

Reply:
Strange, locally this looks like

arch/arm/soc/ti_lm3s6965/CMakeLists.txt    |  2 +

There must be something strange going on with Github.

Reply:
my mistake, rebase was too optimistic, just updated.

Reply:
> There must be something strange going on with Github.

no, it is me, trying to resolve all of those conflicts after the soc move PR  :)

I did a rebase and git decided to make the change with the atmel board for whatever reason.

Reply:
In any case. It is fixed now. Good catch @MaureenHelm .

Reply:
Ah, no, now I made a mistake ...